bigtable/run_hbase_commands.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import subprocess
+
+
+def create_commands_file():
+    file = open("commands", 'w+')
```

Comment:
Nit: maybe `commands.txt` just to make it clear that it's a file. I was confused about `hbase shell commands` below

Reply:
ok